viewReview.py

```diff
@@ -0,0 +1,188 @@
+from flask import request, render_template
+from flask.views import MethodView
+from flask_cas import login_required
+import gbmodel
+
+
+# A method view class that oversees the viewing of individual student reviews
+class ViewReview(MethodView):
+
+    # A function to add a (or some) descriptor word(s) to flush out the numerical ratings we store as answers
```

Comment:
I won't block merging for this but please put these in docstrings: https://www.python.org/dev/peps/pep-0257/